### ISSUE TITLE
Make textfield icons align with textfields

### ISSUE THREAD

**Issue by Fylipp** [Dogfalo/materialize#4822]
### Description
When giving a textfield an icon the vertical icon alignment is off.

### Repro Steps
```html
<div class="input-field">
    <i class="material-icons prefix">vpn_key</i>
    <input id="token" type="text" class="validate" maxlength="10">
    <label for="token">Anmeldecode</label>
</div>
```

### Screenshots / Codepen
https://codepen.io/Fylipp/pen/zzoezO
![image](https://user-images.githubusercontent.com/6892681/27222290-fc1cac8c-528b-11e7-8beb-a230add4fcd7.png)

**Comment by DanielRuf** [comment]
Please try the code from the CSS tab.

https://codepen.io/anon/pen/rwjVba

**Comment by Fylipp** [comment]
That works nicely, why is it not that way by default?

**Comment by DanielRuf** [comment]
@tomscholz Hm, should we pick this or another value?

**Comment by Fylipp** [comment]
I think .5 rem is actually perfect since it is aligned with the label.
![image](https://user-images.githubusercontent.com/6892681/27243682-509261e2-52e3-11e7-9dfe-bec1ae8c811b.png)

**Comment by DanielRuf** [comment]
@Fylipp right, this is what I tested in the ChromeDevTools but I did not test further how other icons react and if some vertical-align or transform makes more sense.

**Comment by manuel-di-iorio** [comment]
The @DanielRuf's solution is perfect!

**Comment by Fylipp** [comment]
@DanielRuf is this issue going to be addressed in the future? I created a [PR](https://github.com/Dogfalo/materialize/pull/5181) that solves the issue the way you proposed.

**Comment by DanielRuf** [comment]
The team will probably merge the PR.

**Comment by acburst** [comment]
Fixed in 3b8cf2d463db63444e73c8157660379ef3f64c6f
